addons/pos_mercado_pago/controllers/main.py

```diff
@@ -0,0 +1,85 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import binascii
+import hashlib
+import hmac
+import logging
+
+from odoo import http
+from odoo.http import request
+
+_logger = logging.getLogger(__name__)
+
+
+class PosMercadoPagoWebhook(http.Controller):
+    @http.route('/pos_mercado_pago/notification', methods=['POST'], type="json", auth="none")
+    def notification(self):
+        """ Process the notification sent by Mercado Pago
+        Notification format is always json
+        """
+        _logger.info('POST message received on the end point')
+
+        # Check for mandatory keys in header
+        x_request_id = request.httprequest.headers.get('X-Request-Id', None)
+        x_signature = request.httprequest.headers.get('X-Signature', None)
+        if not x_request_id or not x_signature:
+            _logger.warning('POST message received with wrong header')
```

Comment:
This is not helpfull to debug, "wrong" but what did you got and what were you expecting?
(to apply to all warning messages below as well)